rts/lra/lra-filters/src/main/resources/META-INF/services/javax.ws.rs.ext.Providers

```diff
@@ -0,0 +1,3 @@
+io.narayana.lra.filter.ClientLRAResponseFilter
```

Comment:
Ok, I have one point. Why is this file needed to be added? By my view this is duplication of the providers registration.

Reply:
I added it back in again. When a request is intercepted via the providers registration the handler for that request may itself may make JAX-RS calls to other services. These latter calls are outgoing and also need filtering and that is what the ClientLRAResponseFilter is for. In other words there are server side and client side filters at play here,

Reply:
I see. Then would it work if you use only the descriptor and not using the `@Provider` annotation to not shown the warning messages during the start?

Reply:
What's the descriptor?